include/seqan3/io/alignment_file/format_sam.hpp

```diff
@@ -381,7 +381,9 @@ class alignment_file_format_sam
      * \param[in]     field_value The value to print.
      */
     template <typename stream_it_t, typename field_type>
-        requires std::ranges::ForwardRange<field_type>
+    //!\cond
+        requires std::ranges::ForwardRange<remove_cvref_t<field_type>>
+    //!\endcond
```

Comment:
why is this change necessary? I would even argue that it is dangerous, because not all ranges are const-iterable. If you pass a range that isn't it will still be accepted by this check, but than fail later on.

Reply:
Does ForwardRange check cbegin/cend?

Reply:
no, why should it?

Reply:
free function cbegin is the same as free function begin with const & argument

Reply:
just need vacation 🍹